src/ngraph/runtime/cpu/cpu_builder.hpp

```diff
@@ -29,6 +29,134 @@
+    else if (R == 2)                                                                               \
+        KV = K<ET, 2>;                                                                             \
+    else if (R == 3)                                                                               \
+        KV = K<ET, 3>;                                                                             \
+    else if (R == 4)                                                                               \
+        KV = K<ET, 4>;                                                                             \
+    else if (R == 5)                                                                               \
+        KV = K<ET, 5>;                                                                             \
+    else if (R == 6)                                                                               \
+        KV = K<ET, 6>;                                                                             \
+    else if (R == 7)                                                                               \
+        KV = K<ET, 7>;                                                                             \
+    else if (R == 8)                                                                               \
+        KV = K<ET, 8>;                                                                             \
+    else if (R == 9)                                                                               \
+        KV = K<ET, 9>;                                                                             \
+    else if (R == 10)                                                                              \
+        KV = K<ET, 10>;                                                                            \
+    else if (R == 11)                                                                              \
+        KV = K<ET, 11>;                                                                            \
+    else if (R == 12)                                                                              \
+        KV = K<ET, 12>;                                                                            \
+    else if (R == 13)                                                                              \
+        KV = K<ET, 13>;                                                                            \
+    else if (R == 14)                                                                              \
+        KV = K<ET, 14>;                                                                            \
+    else if (R == 15)                                                                              \
+        KV = K<ET, 15>;                                                                            \
+    else if (R == 16)                                                                              \
+        KV = K<ET, 16>;
```

Comment:
Why not KV = K<ET, R>?

Reply:
To allow for cases where R is not a constexpr, and the compiler should eliminate these when it is